src/Razor/src/Microsoft.AspNetCore.Razor.Language/Legacy/HtmlMarkupParser.cs

```diff
@@ -1149,8 +1149,6 @@ private MarkupAttributeBlockSyntax ParseRemainingAttribute(MarkupTextLiteralSynt
             var nameContent = string.Concat(name.LiteralTokens.Nodes.Select(s => s.Content));
             if (IsConditionalAttributeName(nameContent))
             {
-                SpanContext.ChunkGenerator = SpanChunkGenerator.Null; // The block chunk generator will render the prefix
```

Comment:
Hmmm, so did this also break in old world?

Reply:
No, this was handled differently in the old world. This line shouldn't have existed in the new world. The comment is a dead give away, we no longer have block chunk generators.